src/rules/checkLineAlignment.js

```diff
@@ -174,6 +186,9 @@ export default iterateJsdoc(({
       {
         additionalProperties: false,
         properties: {
+          customSpacings: {
+            type: 'object',
+          },
```

Comment:
How about we get this child object specified more precisely with:
```js
additionalProperties: false,
properties: {
  postDelimiter: {
    type: 'integer',
  },
  postName: {
    type: 'integer',
  },
  postTag: {
    type: 'integer',
  },
  postType: {
    type: 'integer',
  },
},
```

Reply:
Ah, of course! https://github.com/gajus/eslint-plugin-jsdoc/pull/740/commits/4c973f52c8ee1a156bec02a28b7226ea5ee9eb4a